universal-application-tool-0.0.1/app/views/admin/ProgramListView.java

```diff
@@ -0,0 +1,31 @@
+package views.admin;
+
+import static j2html.TagCreator.body;
+import static j2html.TagCreator.button;
+import static j2html.TagCreator.div;
+import static j2html.TagCreator.each;
+import static j2html.TagCreator.form;
+import static j2html.TagCreator.h1;
+import static j2html.TagCreator.h2;
+import static j2html.TagCreator.p;
+import static j2html.attributes.Attr.FORMACTION;
+
+import com.google.common.collect.ImmutableList;
+import play.twirl.api.Content;
+import services.program.ProgramDefinition;
+import views.BaseHtmlView;
+
+public final class ProgramListView extends BaseHtmlView {
+
+  public Content render(ImmutableList<ProgramDefinition> programs) {
+    return htmlContent(
+        body(
+            h1("Programs"),
+            div(
+                form(
```

Comment:
This doesn't need to be a form. 

Other options:
1. Just use a simple `<a>` and we can style it to look like a button later
2. `<button onclick="window.location.href='...'">Add a Program</button>`